packages/kit/src/core/dev/index.js

```diff
@@ -212,10 +212,14 @@ class Watcher extends EventEmitter {
 										}
 									}
 								}
-
+								let entry = `/${id}`;
+								if (!entry.endsWith('.svelte')) {
+									// TODO utility method?
```

Comment:
I'm not sure the TODO is really necessary
```suggestion
```

Reply:
it was meant as a marker to start a discussion about a more generic solution to the "do we need to append ?import" problem. I'll try to come up with one.

Reply:
removed the todo for now. without additional information from or to vite i don't see a better way. Other than reverting the change in vite that enabled importing .svelte without ?import.